airflow/config_templates/config.yml

```diff
@@ -2639,6 +2639,21 @@ scheduler:
       example: ~
       default: "True"
       see_also: ":ref:`Differences between the two cron timetables`"
+    default_none_schedule:
```

Comment:
I am no sure if we should add new setting in main branch for Airflow 3.

Maybe we should consider having this as an item for the migration tool we want to build? (alert on breaking change for dags without schedule)

Reply:
I believe this one is targetted to be added to 2.10.0 -> so main adding in order to cherry-pick to 2.10.0 is good.

Reply:
but what will happen to this setting in Airflow 3? This PR targets main so I believe this meant to stay in Airflow 3.
My suggestion was not to have it in 3.